scaleway/tests/test_total_count_legacy.py

```diff
@@ -10,6 +10,7 @@
 logger.addHandler(stream_handler)
 
 
+@unittest.skip("To remove")
```

Comment:
Why should this test be removed? Also why is it using unittest instead of pytest?